src/status_im/common/data_confirmation_sheet/view.cljs

```diff
@@ -0,0 +1,80 @@
+(ns status-im.common.data-confirmation-sheet.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.common.data-confirmation-sheet.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- dismiss-keyboard
+  []
+  (rf/dispatch [:dismiss-keyboard]))
+
+(defn- hide-bottom-sheet
```

Comment:
This function could go into `utils.navigation`? Even the var `dismiss-keyboard` could go there. If we feel like `utils.navigation` is not the best name we can change in a separate PR, but just sharing this idea because we've seen these functions been repeatedly redefined in the codebase.